app/api/auth/utils/hexConverter.js

```
@@ -0,0 +1,22 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+var crypto_1 = require("crypto");
+// Function to convert string to 32-byte hex
+function stringToHex(input) {
+    var hash = (0, crypto_1.createHash)('sha256');
+    hash.update(input);
+    return hash.digest('hex').slice(0, 64);
+}
+// Environment variables
+var thirdWebSecretKey = process.env.THIRDWEB_SECRET_KEY;
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)